dev/io.openliberty.faces.internal/bnd.bnd

```diff
@@ -0,0 +1,98 @@
+#*******************************************************************************
+# Copyright (c) 2020 IBM Corporation and others.
+# All rights reserved. This program and the accompanying materials
+# are made available under the terms of the Eclipse Public License v1.0
+# which accompanies this distribution, and is available at
+# http://www.eclipse.org/legal/epl-v10.html
+#
+# Contributors:
+#     IBM Corporation - initial API and implementation
+#*******************************************************************************
+-include= ~../cnf/resources/bnd/bundle.props
+bVersion=1.0
+
+Bundle-Name: JSF Liberty Integration
+Bundle-SymbolicName: io.openliberty.faces.internal
+Implementation-Version: 2.3.6
```

Comment:
is this version correct?

Reply:
Sort of. I'm not sure how to name this bundle because I can't can't just rename the javax imports to jakarta because some of the methods WebContainer.  Those WebContainer methods use` javax.servlet.*` classes because of the interfaces they implement.   

Instead, I use the 2.3.6 project on the build path  and transform this bundle.  It's not used by the 2.3. feature, but it uses the IBM classes from the 2.3 bundle.  I'm talking to Paul about possibly having the jsf 2.3 and jsf 3.0 features using this as a shared bundle.

Reply:
We should get an issue opened so that we can investigate sharing this with the 2.3 project so this is a bit less confusing. As it stands it is a bit hacky since this isn't shared across features currently but there isn't another good way to do this as we've discussed.

Reply:
https://github.com/OpenLiberty/open-liberty/issues/13891